src/Core/Managed/Shared/DataContracts/TelemetryContext.cs

```diff
@@ -121,6 +123,15 @@ public LocationContext Location
             get { return this.properties; }
         }
 
+        /// <summary>
+        /// Gets a Correlation-Context for the operation.
+        /// <see href="https://github.com/lmolkova/correlation/blob/master/http_protocol_proposal_v1.md"/> 
+        /// </summary>
+        internal IDictionary<string, string> CorrelationContext
```

Comment:
let's not expose it now.
It could be confusing for customers and we don't really expect them to use it. Moreover, we encourage them to avoid it.
We can always expose it later, it we have a real need for it.